tests/rt.sh

```diff
@@ -242,7 +242,7 @@ elif [[ $MACHINE_ID = hera.* ]]; then
   QUEUE=batch
   COMPILE_QUEUE=batch
 
-  #ACCNR=fv3-cpu
+  ACCNR=da-cpu
```

Comment:
Please revert these two changes to ACCNR.

Reply:
> Please revert these two changes to ACCNR.

Will do in the final push, and that's why I didn't push it earlier.  Thanks!